src/i18n/en.js

```diff
@@ -0,0 +1,232 @@
+const linkSiteTracking_HTML = `<a href="${urlSiteTracking}">On-Site Tracking</a>`;
+const linkAddYourDomain_HTML = `<a href="${urlSiteTracking}">Add your domain</a>`;
+// TODO: confirm if utm parameter goes here
+const linkPrivacy_HTML = `<a target="_blank" href="${urlPrivacy}?utm_source=app&utm_medium=landing&utm_campaign=signup">Privacy Policy</a>`;
+const linkBuyMonthly_HTML = `<a href="${urlBuyMonthly}">UPGRADE NOW</a>`;
+const linkContactUs_HTML = `<a target="_blank" href="${urlContact}">contact us</a>`;
+
+export default {
+  common: {
+    cancel: `Cancel`,
+    copyright_MD: `
+© ${year} Doppler LLC. All rights reserved. ${linkPrivacyAndLegals_MD}.
+`,
+    help: `Help`,
+    hide: `Hide`,
+    message: `Message`,
+    recaptcha_legal_HTML: `
+Site protected by reCAPTCHA and the Google <a href="https://policies.google.com/privacy?hl=en">Privacy Policy</a>
+and <a href="https://policies.google.com/terms?hl=en">Terms of Service</a>.
+`,
+    send: `Send`,
+    show: `Show`,
+  },
+  feature_panel: {
+    email_automation: `Email automation`,
+    email_automation_description: `Right person, right message, right time`,
+    email_automation_remarks: `Send 100% personalized Emails according to the behavior and interests of your Subscribers. Save time and money!`,
+    email_editor: `Emails Editor`,
+    email_editor_description: `Crea Emails en minutos y accede a nuestra Galería de Plantillas`,
+    email_editor_remarks: `Nuestras Plantillas para Email son totalmente Responsive y fácilmente editables desde nuestro Editor HTML.`,
+    forms: `subscription forms`,
+    forms_description: `Add new contacts to your Lists using custom Forms`,
+    forms_remarks: `Classics and pop-ups with Single or Double Opt-In subscription. You decide how you want them to look, what data to request and where to place them!`,
+  },
+  footer: {
+    iso: `ISO Quality Certification 9001:2008`,
+  },
+  forgot_password: {
+    back_login: `Did you remember your password and want to return?`,
+    confirmation_message_HTML: `
+<p>
+  Check your inbox!
+</p>
+<p>
+  You’ll find an Email with steps to follow.
+</p>
+`,
+    description: `Do not worry! It happens to everyone.`,
+    description2: `Enter your email and we will help you`,
+  },
+  header: {
+    help_url: `${urlHelp}`,
+  },
+  loading: `Loading...`,
+  login: {
+    button_login: `Enter`,
+    enter_doppler: `Enter Doppler`,
+    enter_doppler_sub: `Enjoy the benefits of Email Marketing.`,
+    error_payment: `Account blocked, please contact support.`,
+    forgot_password: `Do not you remember your password?`,
+    head_description: `Attract, engage and convert clients using the Email Marketing Automation power. Try out Doppler!`,
+    head_title: `Free Email Marketing Automation with no sending limits | Doppler`,
+    label_user: `Username: `,
+    signup: `Sign up free`,
+    you_want_create_account: `Do not you have an account yet?`,
+  },
+  reports: {
+    allow_enable_trial_HTML: `
+<p>
```

Comment:
The tabulation here sounds me a little weird. I imagine this tabulated with respect to `allow_enable_trial_HTML`.

Reply:
You are right, as you propose is nicer, and it could work on HTML, but not in Markdown.

Because, since we are using string templates, the spaces are part of the value.

When the value has more than a line, the current approach is: 

1. add a line break before the value
2. indent based on column 0
3. finish with a new line break

![image](https://user-images.githubusercontent.com/1157864/57239914-9e9fdb00-7003-11e9-9ece-72b4b82c30ce.png)

If you like, we could use your proposed indentation for HTML, but not in Markdown, it will be inconsistent. Thoughts?

Reply:
I'm agree to use the identacion for HTML 😄. At least till we replace them with MD.